lib/backend/k8s/resources/globalnetworkpolicies_test.go

```diff
@@ -155,18 +152,13 @@ var _ = Describe("System Network Policies conversion methods", func() {
 		Expect(k).To(Equal(key2))
 	})
 
-	It("should fail to convert an invalid Key to a resource name", func() {
```

Comment:
same as above, we added CIDR to the Spec so we don't need the conversion this test was testing